.github/workflows/android_build.yml

```diff
@@ -219,6 +219,7 @@ jobs:
             $([ -z $GITHUB_TOKEN ] || echo "--config=remote-ci-macos") \
             --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
             --fat_apk_cpu=x86_64 \
+            --define=admin_functionality=enabled \
```

Comment:
Sorry all the code looks good but I'm confused why this is needed given we didn't change the default anywhere?

Reply:
You're right that this isn't needed, this was me trying to fix this error in the Kotlin experimental app:

```
test/kotlin/apps/experimental/MainActivity.kt:58:8: error: unresolved reference: enableAdminInterface
      .enableAdminInterface()
       ^
```

I think the way I'm trying to compile out the Kotlin builder functions isn't working correctly.

Reply:
I figured it out. Reverted this change.